docs/use-cases/wallet.md

```diff
@@ -0,0 +1,140 @@
+
+![](/img/wallet/01.png)
+
+#### 2.1.4 Make Withdrawals (Exits)
+
+Using OMG Network as a value transfer layer offers a faster and cheaper alternative if your work requires frequent operations on the Ethereum network. Most of the time you will hold a certain amount of assets on the OMG Network, and periodically withdraw your funds back to Ethereum. This is the most complicated technical and UX flow. You can implement exits functionality using [Deposit Funds guide](https://docs.omg.network/network/deposits).
+
+*Implementation details:*
+
+1. The entire exit flow for mobile can be very complicated. It involves dealing with challenges, invalid exits, covering standard and in-flight exits. Thus, if you don't have enough time and resources for implementing such functionality, we recommend not to include it in your wallet at all. Exits are perceived as a less frequent activity on a network and can be accomplished with a Web Wallet provided by the OMG Network team.
+2. There's no endpoint to query exit data in the [Watcher Info API](https://docs.omg.network/elixir-omg/docs-ui/?urls.primaryName=master%2Finfo_api_specs). It is available only in [omg-js](https://github.com/omgnetwork/omg-js) library.
+
+### 2.2 Design Guideline
+
+Later 2 mobile UI and UX can be challenging and confusing, thus our job is to make user flows intuitive, appealing, and easy to navigate. Below you can see recommendations from our team.
+
+#### Network Distinction
+
+It's a good practice to have a clear visual distinction between the Ethereum path and the OMG Network path. It's not recommended to display token's balance on both networks together. Setting an obvious separate path helps users to interact with tokens on each network. Additionally, you can set up a switch action (e.g. button) that will help to switch between OMG Network and Ethereum.
+
+![](/img/wallet/02.png)
+
+#### Deposits
+
+1. A user needs to have some ETH in their cryptocurrency wallet to cover the deposit fee. It's recommended to encourage users to deposit fee tokens first and inform about the fee model upfront. This can be mentioned in the tour/onboarding screens or your main screen as follows:
+
+![](/img/wallet/03.png)
+
+2. Deposit is not as frequent operation as transfer, thus you may not add it to the quick actions panel. However, it has to be easily accessible and people should be aware of this functionality through a dedicated menu, intuitive UX flows, or onboarding.
+
+![](/img/wallet/04.png)
+
+3. The OMG Network has a different fee amount for different types of operations, including deposit fee, transfer fee, exit fee, and exit bond. Make sure to be explicit about those, as well as mention if the fee has a flat rate or dynamic rate. You might also set a fee with the highest amount as a default option. Additionally, you may mention that transfer fee is charged in OMG token, the rest of the fees are in ETH.
```

Comment:
>deposit fee, transfer fee, exit fee, and exit bond.

do we have section in the docs that cover each terminology? perhaps in glossary we could point the readers to?

Reply:
I will link to exit bond and general fee explanation. We don't have them covered separately.